src/content/es/2019/performance.md

```diff
@@ -0,0 +1,216 @@
+---
+part_number: II
+chapter_number: 7
+title: Rendimiento
```

Comment:
```suggestion
title: Performance
/```

Reply:
This is probably a bug, but I agree that we should be consistent for now and we can fix this later.